plugins/services/src/js/components/modals/ServiceActionDisabledModal.js

```diff
@@ -363,7 +363,7 @@ class ServiceActionDisabledModal extends React.Component {
   getGroupDeleteMessage() {
     const { actionID, service, intl } = this.props;
     const serviceID = service ? service.getId() : "";
-    const { selectedServices } = this.getServiceTypes(isPackage);
+    const { selectedServices } = this.getServiceTypes(isSDKService);
```

Comment:
@mlunoe - could you provide more details as to why `isSDKService` won't work here? It seems reasonable so I'd like to understand what we're missing. Thanks!